public/index.html

```diff
@@ -95,23 +95,10 @@
     />
 
     <style>
-      .LoadingMessage {
-        position: fixed;
-        top: 0;
-        right: 0;
-        bottom: 0;
-        left: 0;
-        z-index: 999;
-        display: flex;
-        align-items: center;
-        justify-content: center;
-        pointer-events: none;
-      }
-      .LoadingMessage span {
-        background-color: rgba(255, 255, 255, 0.8);
-        border-radius: 5px;
-        padding: 0.8em 1.2em;
-        font-size: 1.3em;
```

Comment:
I should have added a comment here, but this needs to stay in index.html to be styled before the react app loads.

Reply:
Hmm but then this scss is needed in the css file as well as this file won't be used when rendering inside a component. So I can copy the same here as well or I can create a different file and add the stylesheet here. Is that ok?

Reply:
I'm probably missing something, but when is `index.html` not included? Wouldn't that cause other issues, such as not loading fonts?

Reply:
Index.html wont be needed when we use it as an. npm package so i have been testing the umd build which I am creating [here](https://github.com/excalidraw/excalidraw/pull/1455).
So the entry point as you see will be a wrapper and fonts can also be bundled or loaded separately on parent App.
Hence this css I needed in the scss files